ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -276,5 +289,6 @@ export function useGasFeeInputs(
     gasWarnings,
     hasGasErrors,
     supportsEIP1559,
+    updateTransaction,
```

Comment:
Where is this method used?

Reply:
This is used in edit gas fee modal as user make a selection of estimate type. In new designs as user makes a selection of estimate type in edit gas fee modal the modal should close and new estimates should be visible on confirm transaction page.